src/sif/autocost.cc

```diff
@@ -71,6 +71,8 @@ constexpr float kMaxFactor = 100000.0f;
 const std::string kDefaultAutoType = "car";
 constexpr float kDefaultAutoHeight = 1.6f; // Meters (62.9921 inches)
 constexpr float kDefaultAutoWidth = 1.9f;  // Meters (74.8031 inches)
+constexpr float kDefaultAutoLength = 4.0f; // Meters (157,4808 inches)
+constexpr float kDefaultAutoWeight = 2.4f; // / Metric Tons (48,000 lbs)
```

Comment:
that's a huge SUV as default haha

Reply:
SUV, not really. length-wise this is a shorter than normal car. a typical sedan is something like 4.5 to 5m long. weight-wise i would agree this is on the heavier side but if you go with an average sedan weight of somethign like 3500lbs and add 1000lbs of people sitting in it and another 300lbs of crap in the trunk etc you are much closer to 2.4mtons (2.2 mtons).

be careful with the use of german decimal formatting above in the comments, it can really confuse none german speakers/readers

Reply:
Yeah I mean the weight, not the length. And you're right, they don't fit very well.